test/fixtures/resources/ThreatProtection-Attachment/apiproxy/proxies/endpoint1.xml

```diff
@@ -0,0 +1,116 @@
+<ProxyEndpoint name="endpoint1">
+  <Description>Proxy Endpoint 1</Description>
+  <HTTPProxyConnection>
+    <BasePath>/tp-attachment</BasePath>
+    <Properties/>
+    <VirtualHost>secure</VirtualHost>
+  </HTTPProxyConnection>
+
+  <FaultRules/>
```

Comment:
We should include a test case with EV policy in the FaultRule section

Reply:
Done.